data/precip.json

```diff
@@ -0,0 +1 @@
+[{"time":["2016-10-05 01:00:00"],"fips":["12005","12015","12035"],"values":[0,0.04,1]},{"time":["2016-10-05 02:00:00"],"fips":["12005","12015","12035"],"values":[0,0.08,1.3]}] 
```

Comment:
we are going to need to start messing around w/ the animation data soon...any suggestions for formatting @jiwalker-usgs ?